bgpd/bgp_attr_evpn.c

```diff
@@ -36,6 +36,14 @@
 #include "bgpd/bgp_evpn.h"
 #include "bgpd/bgp_evpn_private.h"
 
+bool bgp_route_evpn_cmp(const struct bgp_route_evpn *e1,
+			const struct bgp_route_evpn *e2)
+{
+	return (e1->type == e2->type &&
+		!memcmp(&(e1->eth_s_id), &(e2->eth_s_id), sizeof(esi_t)) &&
+		!ipaddr_cmp(&(e1->gw_ip), &(e2->gw_ip)));
```

Comment:
You can't use `ipaddr_cmp` to compare `union gw_addr`, I think this is what causes troubles in the CI.
Please take a look at my PR https://github.com/FRRouting/frr/pull/10381 – I propose to replace `union gw_addr` with `struct ipaddr`. You can wait until my PR is merged and rebase on top of it to fix CI issues.

Reply:
Hi Igor,
Yes, I noticed that we can't use `ipaddr_cmp`,
that's why I moved this PR to draft stage again.
Sure, Will wait for your changes to get merged.
Thank you :)

Reply:
Fixed.